my_dictionary.txt

```diff
@@ -0,0 +1,289 @@
+personal_ws-1.1 en 0
+acyclic
+afzal
+alertmanager
+allinone
+analytics
+anh
+anonymized
+anonymizer
+ansari
+ansh
+api
+apis
+apl
+ashmita
+async
+auth
+autoscale
+autoscaler
+backend
+backends
+bak
+balancer
+bansal
+bohara
+bootcamp
+cassandra
+checksums
+cli
+clickhouse
+clientstable
+cncf
+cni
+codebase
+compositereporter
+config
+configs
+containerd
+coredns
+cpu
+cqlshrc
+crd
+crds
+cron
+crs
+cryptographic
+daemonset
+datasource
+deliverables
+devops
+dns
+dockerhub
+dockerimages
+downsampling
+dropdown
+dropwizard
+durations
+ebpf
+eg
+eks
+elasticsearch
+emsgsize
+endian
+enums
+env
+equiped
+eweek
+exe
+facto
+fargate
+farro
+featurestable
+filesystem
+firehose
+flink
+fluentd
+frontend
+gaby
+github
+glvs
+gobin
+goland
+golang
+goyal
+grafana
+graphql
+grpc
+haproxy
+hardcoded
+hashicorp
+hasparent
+hcl
+healthcheck
+hostname
+hostnames
+hotrod
+hpa
+httpd
+https
+httpsender
+ideation
+ides
+idl
+ilm
+imagepullsecrets
+incase
+incrementing
+influxdb
+ingester
+ingester's
+ingesters
+ingestor
+init
+initializer
+integrations
+io
+istio
+istio's
+jaeger
+jaeger's
+jaegertracing
+javascript
+jaxrs
+js
+json
+jupyter
+juraci
+kerberos
+keycloak
+keyspace
+kib
+kibana
+kiriti
+kube
+kubernetes
+latencies
+learnings
+lfx
+lifecycle
+loggingreporter
+logz
+lookback
+loopback
+lyft
+macos
+mailgroup
+marck
+mentee
+mentees
+mentorship
```

Comment:
mentorship is not a recognized word?

Reply:
Even I found that to be odd that Aspell's default dictionary doesnt recognise mentorship and some other words. However, when I tried alternatives like misspelling or hunspell I faced a lot more other issues like reporting hashes and hexadecimal strings as wrong spellings, etc. As Aspell has been the best in this regard I thought I'd just add it to the dictionary to make it work (there are not that many words with such false positives as far as I know)

Reply:
There are standard dictionaries in aspell ecosystem, aren't there? We need to use those and only add our dictionary with our specific terms.

Reply:
Yes this dictionary is on top of the standard dictionary offered by the aspell ecosystem.